integration_tests/integration/incidentDetailsEdit.cy.ts

```diff
@@ -277,5 +279,81 @@ context('Incident details (edit) - statement incomplete', () => {
         expect(loc.pathname).to.eq(adjudicationUrls.detailsOfOffence.urls.start(34))
       })
     })
+    it('should check that DICOVERY button to Yes if DISCOVERY date same as INCIDENT date ', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="Yes"]').should('be.checked')
+    })
+    it('should fail to submit as Hours is cleared with appropriate message', () => {
+      cy.visit(adjudicationUrls.incidentDetails.urls.edit('G6415GD', 34))
+      const incidentDetailsPage: IncidentDetails = Page.verifyOnPage(IncidentDetails)
+      incidentDetailsPage.locationSelector().select('Workshop 2')
+      incidentDetailsPage.radioButtonsDiscovery().find('input[value="No"]').click()
+      incidentDetailsPage.timeInputHoursDiscovery().clear()
+      incidentDetailsPage.submitButton().click()
+      incidentDetailsPage
+        .errorSummary()
+        .find('li')
+        .then($errors => {
+          expect($errors.get(0).innerText).to.contain('Enter the time of the discovery')
+        })
+    })
```

Comment:
Good test!
If you're just checking for problems when the discovery time is cleared, you shouldn't really also change the location selector too - it overcomplicates things (and you aren't checking what happens when that's done so it's not necessary) so line 290 where you change location could be removed